Server/src/bgp/OpenMsg.h

```diff
@@ -12,6 +12,7 @@
 
 #include "Logger.h"
 #include "bgp_common.h"
+#include "BMPReader.h"
```

Comment:
Is this needed here?  Looks like it's not and its better to not include in the header file since that can lead to recursion.   

Can your remove this?